docs/REST API/rest-api.md

```diff
@@ -8982,7 +8982,7 @@ Returns information about the specified software. By default, `versions` are sor
       }
     },
     "app_store_app": null,
-    "source": "apps",
+    "source": "apps", #ios_apps,ipados_apps
```

Comment:
should we also add this to the `app_store_app` example below?

Reply:
Added!